### PR TITLE
fix review job location layout for multiple location

### PR DESCRIPTION
**Pull request by alexbowen**
Screenshot 2021-08-09 at 10 03 00

https://dfedigital.atlassian.net/browse/TEVA-2940

### CODE REVIEW

File: app/views/publishers/vacancies/review.html.slim [hunk 1]
```diff
@@ -18,7 +18,7 @@
 
       ol.app-task-list
         - if current_organisation.school_group?
-          li = render "publishers/vacancies/edit_vacancy_sections/edit_job_location"
+          li.mutliline = render "publishers/vacancies/edit_vacancy_sections/edit_job_location"
```
Comment (david-mears-dfe) [inline app/views/publishers/vacancies/review.html.slim:21]: mutli => multi

File: app/components/review_component/review_component.scss [hunk 2]
```diff
@@ -100,3 +100,10 @@
 .action-required-tag {
   @extend .notification-tag;
 }
+
+.mutliline {
```
Comment (david-mears-dfe) [inline app/components/review_component/review_component.scss:104]: mutli => multi

**Review by david-mears-dfe — APPROVED** [review]
Approved if you correct the spelling

File: app/components/review_component/review_component.scss (hunk 2 above)
Comment (alexbowen) [reply app/components/review_component/review_component.scss:104]: ah mutli is the dog from wacky races! @david-mears-dfe

**Pull request merged**